fhir-search/src/main/java/com/ibm/fhir/search/location/NearLocationHandler.java

```diff
@@ -166,53 +168,81 @@ public BoundingBox createBoundingBox(double latitude, double longitude, double d
         List<Bounding> boundingAreas = new ArrayList<>();
         // We are only interested in the near and near-distance parameters.
```

Comment:
my understanding is the `near-distance` went away and is now included as part of the `near` value...maybe update the comment to match?

Reply:
Sure.  no worries, that was a comment I missed removing.